openssl/src/cms.rs

```diff
@@ -97,13 +97,26 @@ impl CmsContentInfoRef {
     }
 
     to_der! {
-    /// Serializes this CmsContentInfo using DER.
-    ///
-    /// OpenSSL documentation at [`i2d_CMS_ContentInfo`]
-    ///
-    /// [`i2d_CMS_ContentInfo`]: https://www.openssl.org/docs/man1.0.2/crypto/i2d_CMS_ContentInfo.html
-    to_der,
-    ffi::i2d_CMS_ContentInfo
+        /// Serializes this CmsContentInfo using DER.
+        ///
+        /// OpenSSL documentation at [`i2d_CMS_ContentInfo`]
+        ///
+        /// [`i2d_CMS_ContentInfo`]: https://www.openssl.org/docs/man1.0.2/crypto/i2d_CMS_ContentInfo.html
+        to_der,
+        ffi::i2d_CMS_ContentInfo
+    }
+}
+
+#[cfg(ossl101)]
```

Comment:
The entire cms module is already `#[cfg(not(libressl))]` so you don't need the extra cfg here.

Reply:
Oh, you're right, I did not notice that.

Reply:
Updated. Now the AppVeyor build fails, but apparently it has nothing to do with the code...